audioread/rawread.py

```diff
@@ -23,6 +23,7 @@
 
 # Produce two-byte (16-bit) output samples.
 TARGET_WIDTH = 2
+PATCH_BYTE = b'\xff'
```

Comment:
The comment above doesn't apply to this constant. So please add a blank line above it and, ideally, write a brief sentence explaining what this is useful for.

Reply:
Really thanks to your advice, I am a fresh graduate, so there must be lots of things to learn lol. The suggestions are quite helpful to me.